extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test/multipart/MultipartBinaryWithoutFilenameTest.java

```diff
@@ -0,0 +1,67 @@
+package io.quarkus.resteasy.reactive.server.test.multipart;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.function.Supplier;
+
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.core.MediaType;
+
+import org.jboss.resteasy.reactive.PartType;
+import org.jboss.resteasy.reactive.RestForm;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+public class MultipartBinaryWithoutFilenameTest {
+
+    @RegisterExtension
+    static QuarkusUnitTest test = new QuarkusUnitTest()
+            .setArchiveProducer(new Supplier<>() {
+                @Override
+                public JavaArchive get() {
+                    return ShrinkWrap.create(JavaArchive.class)
+                            .addClasses(MultipartDataInputTest.Resource.class, MultipartDataInputTest.Item.class,
+                                    MultipartDataInputTest.Result.class);
+                }
+            });
+    private final File IMAGE_FILE = new File("./src/test/resources/image.png");
```

Comment:
I appreciate the nice picture, but given that it's 5MB added to the source archives and history, perhaps we want to generate an arbitrary binary array in the code as a slimmer alternative?

Reply:
Fixed

Reply:
I don't know if something's gone wrong with refreshing this PR, but I don't see the changes :(

Reply:
I changed the image :). From 5MB to 30k :)

Reply:
The Github UI does not agree… 
![Screenshot from 2024-02-12 11-43-23](https://github.com/quarkusio/quarkus/assets/179265/ac7e67b0-c129-45e5-9e4f-aab67dbfd9bc)

Reply:
How about now?